buildfix.sh

```diff
@@ -37,7 +37,11 @@ echo "Formatting WORKSPACE/BUILD files..."
 buildifier -r .
 
 echo "Building and running gofmt..."
-bazel run "${BAZEL_QUIET_FLAGS[@]}" //:gofmt -- -w .
+gofmt_exit_code=$(bazel run "${BAZEL_QUIET_FLAGS[@]}" //:gofmt -- -w . || echo $?)
```

Comment:
to prevent the symlink errors, we could replace this line with

```
mapfile -t GO_SRCS < <(git ls-files '*.go')
bazel run "${BAZEL_QUIET_FLAGS[@]}" //:gofmt -- -w "${GO_SRCS[@]}"
```

Reply:
Cool! Thanks Brandon! 🙏 Macs don't have mapfile, but replaced with something equivalent.

Reply:
I think gofmt is capable of following symlinks and processing them, but the issue in your case is that there was a "dangling" symlink (nbd.proto) which we deleted, and so it no longer exists under the bazel output base. If you had run `git clean -fdx` then bazel build and re-run `sgp` to symlink protos, that would probably have fixed the issue as well.